cassandra-sidecar-release/finish_release.sh

```diff
@@ -0,0 +1,288 @@
+    IFILE=`echo $(basename -- "$i") | cut -c 1`
+    echo $IFILE
+    if [[ $IFILE != "." ]];
+    then
+      FDIR=`echo $i | cut -c ${ROOTLEN}-${#i}`
+      echo "Uploading $FDIR"
+      execute "curl -X PUT -T $i -u${asf_username}:${ARTIFACTORY_API_KEY} https://apache.jfrog.io/artifactory/cassandra/cassandra-sidecar/${FDIR}?override=1"
+      execute "curl -X PUT -T $i -u${asf_username}:${ARTIFACTORY_API_KEY} https://apache.jfrog.io/artifactory/cassandra-deb/cassandra-sidecar/${FDIR}?override=1"
+      sleep 1
+    fi
+done
+cd $tmp_dir
+
+# Remove dist debian directory. Official download location is https://debian.cassandra.apache.org
+echo "Apache Cassandra Sidecar $release debian artifacts" > "_tmp_msg_"
+execute "svn rm -F _tmp_msg_ https://dist.apache.org/repos/dist/release/cassandra/cassandra-sidecar/$release/debian"
+
+#
+# Public deploy the RedHat packages
+#
+
+echo "Deploying redhat packages ..." 1>&3 2>&4
+
+# Upload to ASF jfrog artifactory
+redhat_dist_dir=$tmp_dir/cassandra-dist-$release-redhat
+execute "svn co https://dist.apache.org/repos/dist/release/cassandra/cassandra-sidecar/$release/redhat $redhat_dist_dir"
+[ -e "$redhat_dist_dir" ] || mkdir $redhat_dist_dir # create it for fake mode, to satisfy `find …` command below
+execute "cd $redhat_dist_dir"
+
+ROOTLEN=$(( ${#redhat_dist_dir} + 1))
+
+for i in $(find ${redhat_dist_dir} -mindepth 1 -type f -mtime -10 -not -path "*/.svn/*" -printf "%T@ %p\n" | sort -n -r | cut -d' ' -f 2); do
+    IFILE=`echo $(basename -- "$i") | cut -c 1`
+    if [[ $IFILE != "." ]];
+    then
+        FDIR=`echo $i | cut -c ${ROOTLEN}-${#i}`
+        echo "Uploading $FDIR"
+        execute "curl -X PUT -T $i -u${asf_username}:${ARTIFACTORY_API_KEY} https://apache.jfrog.io/artifactory/cassandra-rpm/cassandra-sidecar/${repo_series}/${FDIR}?override=1"
+        sleep 1
+    fi
+done
+cd $tmp_dir
+
+# Remove dist redhat directory. Official download location is https://redhat.cassandra.apache.org
+echo "Apache Cassandra Sidecar $release redhat artifacts" > "_tmp_msg_"
+execute "svn rm -F _tmp_msg_ https://dist.apache.org/repos/dist/release/cassandra/cassandra-sidecar/$release/redhat"
+
+# Cleaning up
+execute "cd $cassandra_dir"
+rm -rf $tmp_dir
+
+# Restore stdout/stderr (and close temporary descriptors) if not verbose
+[ $verbose -eq 1 ] || exec 1>&3 3>&- 2>&4 4>&-
+
+mail_file="$mail_dir/mail_release_$release"
+[ ! -e "$mail_file" ] || rm $mail_file
+
+echo "[RELEASE] Apache Cassandra Sidecar $release released" > $mail_file
+echo "" >> $mail_file
+echo "The Cassandra team is pleased to announce the release of Apache Sidecar Cassandra version $release." >> $mail_file
+echo "" >> $mail_file
+echo "" >> $mail_file
+echo "Downloads of source and binary distributions are available here:" >> $mail_file
+echo "" >> $mail_file
+echo " https://dlcdn.apache.org/cassandra/cassandra-sidecar/$release/" >> $mail_file
```

Comment:
are there future plans to add the sidecar to the https://cassandra.apache.org/_/download.html page ? 
(this is the preferred approach, but can be done later)